### DISCUSSION TITLE
Change datetime format

### DISCUSSION THREAD

**Discussion by scott-ease** [xenocrat/chyrp-lite#265] [General]
The current format of post publishing date is "day month year". I'd like to change it to "month day, year". I checked all locale folders but couldn't find the right place. Please kindly tell me from which file should I make the change? Thank you.

**Comment by xenocrat** [comment]
Hello there,

You need to change a line in [_content/post.twig_](https://github.com/xenocrat/chyrp-lite/blob/master/themes/blossom/content/post.twig#L23) for your selected theme:

    {{ post.created_at | time("F d Y") }}

The filter function is located [in _Leaf.php_](https://github.com/xenocrat/chyrp-lite/blob/master/includes/lib/Leaf.php#L445), in case you would like to know more about how it works.
